app/models/variant_detail.rb

```diff
@@ -8,6 +8,10 @@ def display_name
     super || variant
   end
 
+  def description
+    super || "#{split.name} = #{variant}"
+  end
```

Comment:
toying with the idea of a default description, but not sure what it should say. seems better than saying nothing, since everything will be blank otherwise to start, and at least for our use case, we're not expecting to go back and add metadata to _every_ previous active test.